### DISCUSSION TITLE
Bouncer AuthorizationException import

### DISCUSSION THREAD

**Discussion by melokki** [adonisjs/core#3224] [Help] [answered]
Hello there
I've started to use `bouncer` in my adonis application and I am wondering, how can one check the `AuthorizationException` that bouncer throws?

I am trying to do something like this:

```js
if (error instanceof AuthorizationException) {
        return response.unauthorized({
        message: 'You are not allowed to view this page',
    })
}
```

But I cannot import the `AuthorizationException` from anywhere.
Is there any method I could do it?

**Comment by thetutlage** [comment, marked as answer]
If you can use the exception code. 

```ts
if(error.code === 'E_AUTHORIZATION_FAILURE') {
  // logic
}
```

**Reply by melokki** [reply]
thank you very much for your answer :pray:
